prometheus-client/src/Prometheus.hs

```diff
@@ -123,6 +123,7 @@ module Prometheus (
 ,   exponentialBuckets
 ,   linearBuckets
 ,   getHistogram
+,   observeWithExemplar
```

Comment:
Could add something like `observeDurationWithExemplar` too, to fully match the existing functions.